api/project.go

```diff
@@ -262,18 +262,21 @@ func FormatTemplate(template string, tokens map[string]string) (string, error) {
 	return template, nil
 }
 
-func buildTemplateExpr(template string, tokens map[string]string) (string, error) {
-	keys, fixed := getTemplateTokens(template)
+// Similar to FormatTemplate, except that it will also escape special regexp characters in the delimiters
+// of the template string.
+func formatTemplateRegexpEscaped(template string, tokens map[string]string) (string, error) {
```

Comment:
I don't feel like the regex is being escaped, but rather the correct regex is being generated

Reply:
`formatTemplateRegexp` would probably be better, not for escaping purposes, but for generating the correct regular expressions